precompiles/author-mapping/src/tests.rs

```diff
@@ -95,7 +95,7 @@ fn selectors() {
 	assert_eq!(Action::UpdateAssociation as u32, 0xd9cef879);
 	assert_eq!(Action::ClearAssociation as u32, 0x7354c91d);
 	assert_eq!(Action::RegisterKeys as u32, 0x4f50accf);
-	assert_eq!(Action::SetKeys as u32, 0xa8259c85);
+	assert_eq!(Action::SetKeys as u32, 0x47f92fc4);
```

Comment:
Breaking change in the precompile side, shall we maintain the previous and make the new one as a new function or just break it and communicate?

Reply:
cc @crystalin @librelois I think the plan was to break compatibility because this extrinsic isn't used by anyone yet

Reply:
Sounds good

Reply:
I agree